{"event": "verbose", "uuid": "58121281-51cd-4d3b-b0f7-08c8f1643dd0", "counter": 6, "stdout": "\u001b[0;31mThe error appears to be in '/home/duchess/Documents/src/github.com/edwardtheharris/ansible-kcp/tmp_dir_sphinx_ansible/temp_file.yaml': line 1, column 3, but may\u001b[0m", "start_line": 5, "end_line": 6, "runner_ident": "5d19a022-1605-4de3-aa59-d396f613a370", "created": "2024-07-23T18:49:14.592182+00:00"}